addons/AD/password_change_notifier.ps1

```diff
@@ -0,0 +1,36 @@
+##########################################################################################
+#Powershell script to unregister disabled Active Directory account based on the UserName.#
+##########################################################################################
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+cls
+$eventTypeID = 4724
+$events = Get-WinEvent -MaxEvents 10  -FilterHashTable @{ Logname = "Security"; ID = $eventTypeID }
+
+$eventArr = @()
+
+foreach ($event in $events)
+{
+    $e = @{ }
+    $xmlObject = [xml]$event.ToXml()
+    $e['recordID'] = $event.RecordId
+    $e['eventTime'] = $event.TimeCreated
+
+    if ($xmlObject.Event.EventData.Data.Length -ge 0)
+    {
+        foreach ($entry in $xmlObject.Event.EventData.Data)
+        {
+            $e[$entry.Name] = $entry.InnerText
+        }
+    }
+    $eventArr += $e
+}
+
+$eventJson = $eventArr | ConvertTo-Json
+$url = "http://@PACKETFENCE_IP:@NTLMAUTH_LISTENING_PORT/password_change"
```

Comment:
the port of ntlm_auth_api is not exposed on the server (iptables prevent that)

Reply:
updated. Please re-review.